src/app/shared/components/template/components/tabs/tabs.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-tab-group
+  [mat-stretch-tabs]="!params().compactButtons"
+  [animationDuration]="animationDuration()"
+>
+  @for (childRow of rows() | filterDisplayComponent; track trackByRow($index, childRow)) {
+    <mat-tab [label]="getTabLabel(childRow)" [attr.data-rowname]="childRow.name">
```

Comment:
nit(non-blocking): alternatively, you could use a `computed` statement on the component level to automatically map the `rows()` to a new data structure that filters to be tab components and also populates the label. This keeps the display component a bit cleaner (doesn't have to call back to the template logic on render).

You could also fully extract the child tab rows during that step and render directly in this component, removing the need for the separate tab component. Something like
```ts
public tabRows = computed(()=>rows()
  .filter(row=>{...})
  .map(tabRow=>({
    label:tabRow.params.label || row.name,
    childRows: tabRow.rows
  })
))
```